aptos-move/framework/aptos-stdlib/sources/cryptography/multi_key.move

```diff
@@ -0,0 +1,42 @@
+module aptos_std::multi_key {
+    use std::hash;
+
+    //
+    // Error codes
+    //
+
+    //
+    // Constants
+    //
+
+    /// The identifier of the MultiEd25519 signature scheme, which is used when deriving Aptos authentication keys by hashing
+    /// it together with an MultiEd25519 public key.
+    const SIGNATURE_SCHEME_ID: u8 = 3;
+
+    /// An *unvalidated*, k out of n MultiKey public key. The `bytes` field contains (1) a vector of single key public keys and
```

Comment:
Should there be a function here that takes n public keys and a threshold k and produce this combined public key? It'd make testing easier. Speaking of which, can we add unit tests for this module?